maths/cube_root.py

```diff
@@ -0,0 +1,10 @@
+
+def cube_root(a: float) ->float:
+    """
+    it just give cube root
+    """
```

Comment:
```suggestion
    >>> cube_root(27)
    3
    """
```

just a little doctest